Superlime.py

```diff
@@ -14,7 +14,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import sublime, sublime_plugin, subprocess, os, tempfile, re, binascii
+import sublime, sublime_lib, sublime_plugin, subprocess, os, tempfile, re, binascii
```

Comment:
To go with this import, you need to add a `dependencies.json` file as described in `sublime_lib`'s [installation instructions] (https://github.com/SublimeText/sublime_lib?tab=readme-ov-file#installation).